File: Java/01 - Introduction to Programming/Task14/task1.java
import java.util.*;

public class task1
{
    public static void main(String[] args)
    {
        Scanner keyboardInput = new Scanner(System.in);
        
        //request input from user
        System.out.print("Please enter a number: ");
        int num = Integer.parseInt(keyboardInput.nextLine());
        int[] range = {1,2,3,4,5,6,7,8,9,10,11,12};

        for (int i : range)
        {
            System.out.println(num + " x " + i + " =  " + num*i);
        }

        keyboardInput.close();
    }
}

File: Ruby/01 - Introduction to Programming/Task14/task1.rb
#Task14 - task1.rb
#------------------------------------------------------------#
#request input from user
print "Please enter a number: "
num = gets.chomp().to_i

for i in Range.new(1, 12)
	puts num.to_s + " x " + i.to_s + " =  " + (num*(i)).to_s
end